src/utils/psql.ts

```diff
@@ -0,0 +1,20 @@
+import { log } from '../log.js';
+
+export const psql = async (
+  connection_uri: string,
+) => {
+  const { execSync, spawnSync } = await import('child_process');
```

Comment:
I'm not sure if it's ok to use a synchronous version of `exec` and `spaws`, but it works.